eZ/Publish/Core/Persistence/Legacy/Content/Location/Gateway/DoctrineDatabase.php

```diff
@@ -631,6 +657,30 @@ function ($pathString) use ($query, $handler) {
         $query->prepare()->execute();
     }
 
+    private function buildHiddenSubtreeQuery(string $selectExpr): QueryBuilder
+    {
+        $query = $this->connection->createQueryBuilder();
+        $expr = $query->expr();
+        $query
+            ->select($selectExpr)
+            ->from('ezcontentobject_tree', 't')
+            ->leftJoin('t', 'ezcontentobject', 'c', 't.contentobject_id = c.id')
+            ->where(
+                $expr->orX(
+                    $expr->eq(
+                        't.is_hidden',
+                        $query->createPositionalParameter(1, PDO::PARAM_INT)
+                    ),
+                    $expr->eq(
+                        'c.is_hidden',
+                        $query->createPositionalParameter(1, PDO::PARAM_INT)
```

Comment:
It is actually not int, but boolean in this case. Might crash on some PosgreSQL versions in the future.

Reply:
Fixed in https://github.com/ezsystems/ezpublish-kernel/pull/3142/commits/66ee6a667394b4bb962d330baef104e4e07ef042